### PR TITLE
remove onsceneload, set indicator when game start

### PR DESCRIPTION
**Pull request by lxymahatma**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,353 @@
-.idea
```
Comment (Azn9) [inline .gitignore:1]: Could you leave this line? I use Jetbrains Rider 😅

File: AccuracyIndicator/Indicator/InGameIndicator.cs [hunk 2]
```diff
@@ -39,11 +39,11 @@ private void Start()
         _canvas.AddComponent<GraphicRaycaster>();
         _canvas.GetComponent<Canvas>().renderMode = RenderMode.ScreenSpaceOverlay;
 
-        // -0.14 | -0.05 | -0.025 | 0.025 | 0.05 | 0.14
+        // -0.13 | -0.05 | -0.025 | 0.025 | 0.05 | 0.13
```
Comment (Azn9) [inline AccuracyIndicator/Indicator/InGameIndicator.cs:42]: Not sure if we really should restrict to 0.13, I managed to get -0.135 (by smashing my keyboard, but I reached this value)
But if you really want it, you should also update the bar widths to fit (and the position of the hit markers)

**Review by Azn9 — CHANGES_REQUESTED** [review]
I didn't know the `GameStart` method exists (I haven't searched much), thanks!

File: .gitignore (hunk 1 above)
Comment (lxymahatma) [reply .gitignore:1]: oh ok lol, it's auto generated by vs

File: AccuracyIndicator/Indicator/InGameIndicator.cs (hunk 2 above)
Comment (lxymahatma) [reply AccuracyIndicator/Indicator/InGameIndicator.cs:42]: well from the aspect that the game set it as 0.13 ig we should use it? to make it more accurate from the game side

Comment (Azn9) [reply AccuracyIndicator/Indicator/InGameIndicator.cs:42]: Okay! Then we'll need to update the widths of the different things. Would you like to do it or do I?

Comment (lxymahatma) [reply AccuracyIndicator/Indicator/InGameIndicator.cs:42]: yeah i didnt look detailed about these code (also not that understand bc my low level) so i think it's better for you to do this 👍

**Pull request merged**